custom/ceea/ceeafacades/src/com/ceea/facades/populators/CeeaPLPPopulator.java

```diff
@@ -0,0 +1,97 @@
+/**
+ *
+ */
+package com.ceea.facades.populators;
+
+import de.hybris.platform.commercefacades.product.data.ImageData;
+import de.hybris.platform.commercefacades.product.data.ProductData;
+import de.hybris.platform.commercefacades.search.converters.populator.SearchResultVariantProductPopulator;
+import de.hybris.platform.commerceservices.search.resultdata.SearchResultValueData;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.commons.lang.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.ceea.facades.product.data.AnnotationData;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+
+/**
+ * @author AEN8KOR
+ *
+ */
+public class CeeaPLPPopulator extends SearchResultVariantProductPopulator
+{
+
+	private static final Logger LOG = LoggerFactory.getLogger(CeeaPLPPopulator.class);
+
+	@Override
+	public void populate(final SearchResultValueData source, final ProductData target)
+	{
+		super.populate(source, target);
+
+		target.setAnnotation(this.<String> getValue(source, "annotation"));
+		target.setCeeaThreeDimensionalImage(createImageDataObj(source));
+		target.setPrimaryImgURL(this.<String> getValue(source, "primaryImgURL_string"));
+		try
+		{
+			convertJsonToObject(target);
+		}
+		catch (final IOException e)
+		{
+			LOG.info("Exception Occured :: " + e.getMessage());
+		}
+	}
+
+	/**
+	 * @param target
+	 */
+	private void convertJsonToObject(final ProductData target) throws IOException
```

Comment:
Add this method into util as same code is being used on plp and pdp

Reply:
Add this into util and this isuse is resolved.